packages/schema/src/Schema.ts

```diff
@@ -6219,11 +6204,13 @@ export interface Class<Self, Fields extends Struct.Fields, A, I, R, C, Inherited
     disableValidation?: boolean | undefined
```

Comment:
line 6203
> props: keyof C extends never ? void | {} : C,

props with defaults need to be subtracted, because you can create without props when the required ones have defaults.